modules/store/src/index.ts

```diff
@@ -6,6 +6,7 @@ export {
   MetaReducer,
   Selector,
   SelectorWithProps,
+  RuntimeChecks,
```

Comment:
These don't need to be exported publicly except for `USER_PROVIDED_META_REDUCERS`